docroot/themes/custom/mass_theme/templates/content/node--topic-page.html.twig

```diff
@@ -67,20 +67,18 @@
           } %}
         {% endfor %}
       {% endblock %}
-      {# {% block secondaryRelationship %}
-        {% for item in secondary %}
-          {% include "@atoms/item-link.twig" with {
-            "itemLink": {
-              "title": item.title,
-              "href": item.href
-            }
-          } %}
-        {% endfor %}
-      {% endblock %} #}
     {% endembed %}
   {% endif %}
-
-  {% include "@organisms/page-banner/page-banner.twig" %}
+  {% include "@organisms/page-banner/page-banner.twig" with {
+    "pageBanner": {
+      "theme": "c-white",
+      "pageHeader": {
+        "title": node.label,
+        "underline": "TRUE",
+        "description": content.field_topic_lede
```

Comment:
I think in production we are hiding all the topic descriptions. I don't see any descriptions in prod, so we can remove this line or comment it out
prod: https://www.mass.gov/topics/early-childhood-education-care
tugboat: https://pr2266-6u43mqlobnreiawjeqltk8csy9w6so8y.tugboatqa.com/topics/early-childhood-education-care

Reply:
Done.